apps/dashboard/src/lib/features/course/pages/people.svelte

```
@@ -68,24 +118,48 @@
     return profile ? profile.email : email;
   }
 
-  function obscureEmail(email) {
-    const [username, domain] = email.split('@');
-    const obscuredUsername =
-      username.charAt(0) + '*'.repeat(username.length - 2) + username.charAt(username.length - 1);
+  function gotoPerson(person: CourseMember) {
+    if (!person.profileId) return;
 
-    return `${obscuredUsername}@${domain}`;
+    goto(`${page.url.href}/${person.profileId}`);
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)